docs/diagrams/UndoSequenceDiagram-Logic.puml

```diff
@@ -24,7 +24,7 @@ activate UndoCommand
 UndoCommand --> AddressBookParser
 deactivate UndoCommand
 
-AddressBookParser --> LogicManager : u
+AddressBookParser --> LogicManager :
 deactivate AddressBookParser
 
 LogicManager -> UndoCommand : execute()
```

Comment:
Can we delete everything related to undo? The undo puml files still show addressbook too

Reply:
Got it